sql/core/src/main/scala/org/apache/spark/sql/execution/Generate.scala

```diff
@@ -34,24 +35,12 @@ import org.apache.spark.sql.catalyst.expressions._
 @DeveloperApi
 case class Generate(
     generator: Generator,
+    output: Seq[Attribute],
```

Comment:
It seems redundant to have the generator `Attribute`s in both the operator and the expression.